common/src/main/java/org/apache/uniffle/common/security/HadoopSecurityContext.java

```diff
@@ -119,6 +120,12 @@ private <T> T executeWithUgiWrapper(UserGroupInformation ugi, Callable<T> callab
     return ugi.doAs((PrivilegedExceptionAction<T>) callable::call);
   }
 
+  // Only for tests
+  @VisibleForTesting
+  public Map<String, UserGroupInformation> getProxyUserUgiPool() {
```

Comment:
Could we remove `public`? Test code is in the same package.

Reply:
Done